src/chapters/classes/constructors.rst

```diff
@@ -0,0 +1,157 @@
+Constructors
+============
+
+We’ll often want to initialize, or set the initial value of, some of our
+fields when creating a new object from a class. **Constructors** allow us to do so.
+
+In Java, constructors have the same name as the class and are most often
+declared public (though they can have any other valid access modifier).
+They are declared *without a return type*. Any function that is named
+the same as the class and has no return type is a constructor.
+
+Here is an example of a constructor definition within the ``HelloWorld``
+class:
+
+.. sourcecode:: java
+   :linenos:
+
+   public class HelloWorld {
```

Comment:
what if we also put this guy in the java-web-dev-exercises repo? as something like, helloClasses

Reply:
Sounds good!